clap.zig

```diff
@@ -912,7 +912,7 @@ fn Arguments(
 
         const T = ParamType(Id, param, value_parsers);
         const default_value = switch (param.takes_value) {
-            .none => false,
+            .none => @as(u8, 0),
```

Comment:
Being able to crash a program by passing `256` `-h` seems realistic and undesirable. Let's use a `usize`.

Reply:
Would you prefer making the count a `usize` over changing the increment to use `+|=`?

Reply:
Hmmm. I don't think I really have a preference.